kpatch-build/kpatch-build

```diff
@@ -395,16 +408,27 @@ while [[ $# -gt 0 ]]; do
 		echo "WARNING: Skipping gcc version matching check (not recommended)"
 		SKIPGCCCHECK=1
 		;;
-	--)
-		[[ -z "$2" ]] && die "no patch file specified"
-		[[ ! -f "$2" ]] && die "patch file '$2' not found"
-		PATCHFILE=$(readlink -f "$2")
-		break
+	*)
+		[[ "$1" = "--" ]] && shift && continue
+		[[ ! -f "$1" ]] && die "patch file '$1' not found"
+		PATCH_LIST+=($(readlink -f "$1"))
 		;;
 	esac
 	shift
 done
 
+if [[ ${#PATCH_LIST[@]} = 0 ]]; then
+        PATCH_LIST=(*.patch)
```

Comment:
So if no patches are specified, I think this just uses the patches in the current directory?  If so, I'd rather not go there, it could create some "surprises".

Reply:
Agreed, I'll dump a "no patches specified" msg instead.